### ISSUE TITLE
Feasible to exclude test data (private keys) when packaging Gem?

### ISSUE THREAD

**Issue by JamesJJ** [excon/excon#702]
Our security scanning tools alert about unexpected private key files, with open permissions:

```
./usr/local/bundle/gems/excon-0.64.0/tests/data/127.0.0.1.cert.key
./usr/local/bundle/gems/excon-0.64.0/tests/data/excon.cert.key
```

Is it feasible to exclude the testing data [or just those private keys] when packaging the Gem?

Many thanks ^_^

**Comment by geemus** [comment]
Ok, I took a stab at it. Could you see if this still works and gets you what you need? https://github.com/excon/excon/commit/d278933afae9e91a2d2b8f8ff7a29cb1bb89e47a

**Comment by geemus** [comment]
And also this: https://github.com/excon/excon/commit/e226dad7a55003b5f59aec282cf4e2a10c0cb4bb (adds bundled cert back in that is used normally, but not the test certs).

**Comment by JamesJJ** [comment]
Many thanks @geemus .

Those changes resolve the original concern, and everything seems to work as expected.

((I'm using `excon` as a dependency of [Fluentd's elasticsearch output plugin](https://github.com/uken/fluent-plugin-elasticsearch/blob/master/fluent-plugin-elasticsearch.gemspec#L22). Fluentd running with updated `excon` successfully performs HTTP operations to elasticsearch))

**Comment by geemus** [comment]
Thanks for double checking, I was pretty confident, but didn't have any easy way to ensure your security scanning would be satisfied (especially since there are still bundled certs, they just aren't private ones). Sounds like this gets us what we need though, and as a bonus should actually reduce the gem file size a bit. Released in v0.65.0
